From e29ce24eba59c13099ad8077d04a9276d68a1d75 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Sun, 17 Feb 2013 15:01:44 +0800
Subject: [PATCH 494/666] dts/t4240qds: drop the interrupts property of rtc
 device node

According to the t4240qds schematics, the INT pin of DS3232 doesn't
route to the MPIC.

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t4240qds.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/powerpc/boot/dts/fsl/t4240qds.dts b/arch/powerpc/boot/dts/fsl/t4240qds.dts
index 021e1d9..16eca24 100644
--- a/arch/powerpc/boot/dts/fsl/t4240qds.dts
+++ b/arch/powerpc/boot/dts/fsl/t4240qds.dts
@@ -394,7 +394,6 @@
 					rtc@68 {
 						compatible = "dallas,ds3232";
 						reg = <0x68>;
-						interrupts = <0x1 0x1 0 0>;
 					};
 				};
 
-- 
2.7.4

